plugins/janus_videoroom.c

```diff
@@ -2511,7 +2512,16 @@ int janus_videoroom_init(janus_callbacks *callback, const char *config_path)
 			dynamic_url_status = TRUE;
 			JANUS_LOG(LOG_INFO, "RTSP Dynamic URL is enabled\n");
 		}
-
+		janus_config_item *use_ipv6_val = janus_config_get(config, config_general, janus_config_type_item, "use_ipv6");
+		if (use_ipv6 && !janus_is_true(use_ipv6_val->value))
```

Comment:
use_ipv6 is set to false so it will go to line 2522 and become true and it always be IP6